pkg/testfixture/app_cr.go

```diff
@@ -16,7 +16,7 @@ func NewAppCRFromFilled(modifyFunc func(*applicationv1alpha1.App)) *applicationv
 		},
 		Spec: applicationv1alpha1.AppSpec{
 			Catalog: "test-spec-catalog",
-			Config: applicationv1alpha1.AppSpecConfig{
+			Config: &applicationv1alpha1.AppSpecConfig{
```

Comment:
These pointers still worry me. As I think it can cause bugs due to the nil handling. But no blocker I'd just like to discuss again next week.

Reply:
Agreed. I just want to push the vendor.